src/main/java/com/pirimid/cryptotrade/service/Impl/OrderServiceImpl.java

```diff
@@ -23,22 +34,62 @@
     @Autowired
     AccountRepository accountRepository;
     @Autowired
+    ExchangeRepository exchangeRepository;
+    @Autowired
     UserRepository userRepository;
     @Autowired
+    TradeRepository tradeRepository;
+    @Autowired
     ExchangeUtil exchangeUtil;
     @Autowired
     User user;
+
+    private Order orderResDtoToOrder(OrderResDTO orderDto,Account account){
+        Order newOrder = new Order();
```

Comment:
Let's use constructor to set the value, it will avoid unnecessary code

Reply:
okay we will move this to constructor